docs/source/gs-processing/developer/input-configuration.rst

```diff
@@ -447,6 +447,15 @@ arguments.
         will be considered as an array. For Parquet files, if the input type is ArrayType(StringType()), then the
         separator is ignored; if it is StringType(), it will apply same logic as in CSV.
 
+-  ``huggingface``
+
+   -  Transforms a text feature column to tokens or embeddings with different Hugging Face models, enabling nuanced understanding and processing of natural language data.
+   -  ``kwargs``:
+
+      - ``action`` (String, required): The action to perform on the text data. Currently we only support text tokenization through HuggingFace models, so the only accepted value here is "tokenize_hf".
```

Comment:
Can we add some more detail about what the `tokenize_hf` action does in a sub-bullet here? Imagine it's a use seeing this transformation for the first time, what do they need to know to use it (and what output to expect).